app/Exports/CirclesExport.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Exports;
+
+use App\Eloquents\Circle;
+use Maatwebsite\Excel\Concerns\FromCollection;
+use Maatwebsite\Excel\Concerns\WithHeadings;
+use Maatwebsite\Excel\Concerns\WithMapping;
+
+class CirclesExport implements FromCollection, WithHeadings, WithMapping
+{
+    /**
+    * @return \Illuminate\Support\Collection
+    */
+    public function collection()
+    {
+        return Circle::submitted()->with('leader', 'tags')->get();
+    }
+
+    /**
+    * @var Circle $circle
+    */
+    public function map($circle): array
+    {
+        $tags = [];
+
+        foreach ($circle->tags->all() as $tag) {
+            $tags[] = $tag->name;
+        }
+
+        $leader = $circle->leader->first();
+
+        if ($circle->status === 'approved') {
+            $status = '受理';
+        } elseif ($circle->status === 'rejected') {
+            $status = '不受理';
+        } else {
+            $status = '確認中';
+        }
+
+        return [
+            $circle->id,
+            $circle->name,
+            $circle->name_yomi,
+            $circle->group_name,
+            $circle->group_name_yomi,
+            join(',', $tags),
```

Comment:
```suggestion
            $circle->tags->implode('name', ','),
```

↑こう書くとスマートで良さそう（動作未確認）

![image](https://user-images.githubusercontent.com/6687653/84372202-bb9ada80-ac15-11ea-97c7-d87c6b6d162f.png)